_posts/2019-08-27-inception-pipelines-pt6.markdown

```diff
@@ -0,0 +1,211 @@
+
+![Enter User Details]({{ site.url }}/img/inception-pipelines/part-6/step3-02-UserDetails.png)
+
+Click the `Attach existing policies directly` button and search for the policy we created in the previous section.
+
+![Set Permissions]({{ site.url }}/img/inception-pipelines/part-6/step3-03-SetPermissions.png)
+
+Review the user details.
+
+![Review]({{ site.url }}/img/inception-pipelines/part-6/step3-04-Review.png)
+
+Download and/or copy the credentials for late use in Step 3.
+
+![Credentials]({{ site.url }}/img/inception-pipelines/part-6/step3-05-Credentials.png)
+
+Success!
+
+![Success]({{ site.url }}/img/inception-pipelines/part-6/step3-06-Success.png)
+
+### <a name="awscliisland"></a>AWS CLI Island
+
+On this path you will use the AWS CLI to create an [IAM User](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_users.html) and [Customer Managed IAM Policy](https://docs.aws.amazon.com/IAM/latest/UserGuide/access_policies_managed-vs-inline.html#customer-managed-policies). The commands to execute are below. **Note:** You will need to replace the `111111111111` with your actual AWS Account number.
+
+```bash
+aws iam create-policy --policy-name InceptionPipelineSeedingPolicy --description "Minimum permissions required to seed an Inception Pipeline, and optionally delete it if required." --policy-document https://raw.githubusercontent.com/MechanicalRock/InceptionPipeline/post/part-6/policy.json
+aws iam create-user --user-name InceptionPipelineSeedingUser
+aws iam attach-user-policy --user-name InceptionPipelineSeedingUser --policy-arn "arn:aws:iam::111111111111:policy/InceptionPipelineSeedingPolicy"
+aws iam create-access-key --user-name InceptionPipelineSeedingUser
+```
+
+Because I'm feeling super nice and generous I've wrapped these commands up into a tidy little shell script [here on GitHub](https://github.com/MechanicalRock/InceptionPipeline/blob/post/part-6/seed-inception-pipeline-user.sh). When you run it, this is what you'll see:
+
+```bash
+AWS Account: 111111111111
+AWS CLI profile (default is 'default'): default
+{
+    "Policy": {
+        "DefaultVersionId": "v1",
+        "AttachmentCount": 0,
+        "PolicyId": "ANPAPAPERSCISSORSROCKS",
+        "IsAttachable": true,
+        "UpdateDate": "2018-08-23T02:08:18Z",
+        "Arn": "arn:aws:iam::111111111111:policy/InceptionPipelineSeedingPolicy",
+        "Path": "/",
+        "CreateDate": "2018-08-23T02:08:18Z",
+        "PolicyName": "InceptionPipelineSeedingPolicy"
+    }
+}
+{
+    "User": {
+        "Arn": "arn:aws:iam::111111111111:user/InceptionPipelineSeedingUser",
+        "Path": "/",
+        "UserName": "InceptionPipelineSeedingUser",
+        "UserId": "AIDAADAFORTRANCOBOLCC",
+        "CreateDate": "2018-08-23T02:08:32Z"
+    }
+}
+{
+    "AccessKey": {
+        "CreateDate": "2018-08-23T02:08:56Z",
+        "AccessKeyId": "AKIADEADBEEFBEERGERS",
+        "Status": "Active",
+        "SecretAccessKey": "SecretSquirrelBusinessSsshDontTellASeoul",
```

Comment:
`Soul` not `Seoul`, unless you have something against 🇰🇷 .  Other places further down.

Or `DontTellAnyone` for the less spiritually inclined.

Reply:
Soul was one letter short for the length of a secret key... changed it to DontTellAnyOne.